README.md

```diff
@@ -4,128 +4,114 @@
-3. Install Python 2.7 (Kango requires 2.7 specifically) and put it in your path so it's accessible with either `python` or `python2`. You can manage different Python versions with [pyenv](https://github.com/yyuu/pyenv).
-4. Ensure command line developer tools are installed `xcode-select --install`.
-5. Run `yarn install` within the folder you cloned. This will install all the dependencies needed for the project.
-6. Run `./build` from within the folder you cloned which will build the toolkit, or run `./watch` which will monitor the project directory for changes and build automatically.
-7. Finished extensions for each platform are available in the `output` directory.
-8. Load it into Chrome as an unpacked extension, load it into Firefox via the .xpi file, or load it into Safari using the extension builder (Mac only).
-
-**Windows**
-
-1. Clone the repository.
-2. Install [node](https://nodejs.org/en/download/) and [yarn](https://yarnpkg.com/en/docs/install#windows-tab) (tested with node-v6.9.1 and yarn-v0.24.6). You can manage different versions of Node with [nvm](https://github.com/creationix/nvm).
-3. Install Python 2.7 (Kango requires 2.7 specifically) in your path so it's accessible with just plain `python`.
+2. Install `node` and `yarn` (both are available through `brew`, test with node-v9.2.1 and yarn-1.3.2)
+3. Ensure command line developer tools are installed `xcode-select --install`.
 4. Run `yarn install` within the folder you cloned. This will install all the dependencies needed for the project.
-5. Run `./build` from within the folder you cloned which will build the toolkit, or run `./watch` which will monitor the project directory for changes and build automatically.
-6. Finished extensions for each platform are available in the `output` directory.
-7. Load it into Chrome as an unpacked extension, load it into Firefox via the .xpi file.
+5. Run `yarn build` from within the folder you cloned which will build the Toolkit.
+  - Whilst developing, you may prefer to run `yarn watch` which will monitor the project
+    directory for changes and build automatically.
+6. The built extension will be available in the `dist` folder and is the folder you should
+  select when loading the extension into the browser.
+8. Load it into Chrome as an unpacked extension, load it into Firefox via the .xpi file,
+  or load it into Safari using the extension builder (Mac only).
 
 # Development Methodology
 
 #### ES2015? What's that?
-We've decided that it's better to use the latest and greatest than to wait for browsers to support all the nice new syntax of ES2015. If you're uncomfortable with the new syntax, feel free to use standard Javascript syntax from days of yore. It still works.
+We've decided that it's better to use the latest and greatest than to wait for browsers to
+support all the nice new syntax of ES2015. If you're uncomfortable with the new syntax,
+feel free to use standard Javascript syntax from days of yore. It still works.
```

Comment:
This is all less relevant now that we're forcing class syntax. We can just remove this I think.